tox.ini

```diff
@@ -1,5 +1,8 @@
 [tox]
-envlist = py27, py34, py35, py36
+envlist =
+    py27, py34, py35, py36
+skip_missing_interpreters =
+    true
```

Comment:
this is also used in CI, so i'd rather not default this setting to true. maybe the docs can just suggest using this flag locally for development?